src/test/java/stepdefinitions/StepDefinitions.java

```diff
@@ -1,65 +1,109 @@
 package stepdefinitions;
 
 import io.cucumber.java.PendingException;
+import io.cucumber.java.bs.A;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import org.openqa.selenium.WebDriver;
 import pages.ContactUsPage;
 import pages.MobileChapterPage;
+import pages.Page;
 import pages.ValoriHomePage;
 
 import java.util.ArrayList;
 
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
+import static org.assertj.core.api.Assertions.*;
 
 public class StepDefinitions {
     private WebDriver driver;
+    private ValoriHomePage valoriHomePage;
+    private ContactUsPage contactUsPage;
+    private MobileChapterPage mobileChapterPage;
+    private Page page;
 
-    public StepDefinitions (){
+    public StepDefinitions() {
         this.driver = DriverManager.driver;
+        initPageObjects();
     }
 
+    public void initPageObjects() {
+        valoriHomePage = new ValoriHomePage(driver);
+        contactUsPage = new ContactUsPage(driver);
+        mobileChapterPage = new MobileChapterPage(driver);
+        page = new Page(driver);
```

Comment:
Je zou niet je basepage hoeven initialiseren. Dit moet jij wel omdat je je basepage alleen geimplementeerd hebt in de ContactUsPage. Je bent halverwege je implementatie van een basepage gestopt.